apps/aecore/src/aec_target.erl

```diff
@@ -93,7 +93,8 @@ recalculate(Top, PrevHeaders) ->
     DesiredTimeBetweenBlocks = aec_governance:expected_block_mine_rate(),
     Last                     = lists:last(PrevHeaders),
     TotalTime                = mining_time_between(Last, Top),
-    aec_pow:integer_to_scientific(TotalTime * K div (DesiredTimeBetweenBlocks * SumKDivTargets)).
+    NewTargetInt             = TotalTime * K div (DesiredTimeBetweenBlocks * SumKDivTargets),
+    min(?HIGHEST_TARGET_SCI, aec_pow:integer_to_scientific(NewTargetInt)).
```

Comment:
I am considering whether it compared to (the debatebly more readable) `aec_pow:integer_to_scientific(min(?HIGHEST_TARGET_INT, NewTargetInt))`.

Reply:
I see they are equivalent as scientific notation.